### ISSUE TITLE
[Accessibility] Emoji button couldn't be navigated with tab

### ISSUE THREAD

**Issue by roman-kulakov** [missive/emoji-mart#604]
Hi, due the emoji button has `tabindex=-1` it is skipped by the tab navigation and thus screenreaders couldn't read it, it would be good to have possibility to configure that or maybe add possibility to provide some custom attributes to override:

```
<button
          aria-label={native}
          aria-selected={selected || undefined}
          aria-posinset={posinset}
          aria-setsize={grid.setsize}
          data-keyboard={this.state.keyboard}
          title={this.props.previewPosition == 'none' ? emoji.name : undefined}
          type="button"
          class="flex flex-center flex-middle"
          tabindex="-1"
          onClick={() => this.handleEmojiClick({ emoji })}
          onMouseEnter={() => this.handleEmojiOver(pos)}
          onMouseLeave={() => this.handleEmojiOver()}
          style={{
            width: this.props.emojiButtonSize,
            height: this.props.emojiButtonSize,
            fontSize: this.props.emojiSize,
            lineHeight: 0,
          }}
         {...this.props.emojiButtonAttrs}
>
```

**Comment by CSFlorin** [comment]
Same issue here!

**Comment by CSFlorin** [comment]
@roman-kulakov I was looking through the source and realized this: https://github.com/missive/emoji-mart/blob/main/packages/emoji-mart/src/components/Picker/Picker.js#L280

Turns out it's intended to be navigated with the arrow keys. A little odd but at least keyboard users aren't completely stuck with the right messaging.

**Comment by roman-kulakov** [comment]
@CSFlorin yes, it could be navigated through arrow keys, but it gives us only a keyboard accessibility but not screenreaders accessibility :(

**Comment by Charca** [comment]
This PR doesn't make emojis navigable via tab, but it does make them accessible to screen readers https://github.com/missive/emoji-mart/pull/666

**Comment by EtienneLem** [comment]
Emojis can’t be focused on purpose, because we need the focus to stay in the input. That being said, with #666, I would consider this issue fixed in **v5.2.1**

**Comment by vietanhtran16** [comment]
Hey @EtienneLem could you elaborate on why emojis cant be focused on purpose? It doesnt feel intuitive to me. Compare to slack emoji picker, you can tab through the emojis which feels a lot more natural
